src/cljs/nnts2/routes.cljs

```diff
@@ -2,26 +2,21 @@
   (:require-macros [secretary.core :refer [defroute]])
   (:import goog.History)
   (:require
-    [pushy.core :as pushy]
-    [secretary.core :as secretary]
-    [goog.events :as gevents]
-    [goog.history.EventType :as EventType]
-    [re-frame.core :as re-frame]
-    [nnts2.events :as events]))
+   [pushy.core :as pushy]
+   [secretary.core :as secretary]
+   [goog.events :as gevents]
+   [goog.history.EventType :as EventType]
+   [re-frame.core :as re-frame]
+   [nnts2.events :as events]))
 
 (enable-console-print!)
 
 (defroute "/home" []
-  (do
-    (re-frame/dispatch [::events/set-active-panel :home-panel])
-    (re-frame/dispatch [:nnts2.user.events/get-user-info])))
+  (re-frame/dispatch [::events/set-active-panel :home-panel]))
```

Comment:
@sezaljain we should talk about event naming sometime, do remind me when I'm in office.

The event here is "The home route was navigated to."
"Set the active panel to the home panel" is not the event, it is the _action_ to be taken in response to the event. 

Naming the events appropriately reduces coupling between the views and the event handlers. It's worth discussing these tradeoffs.